Docs/imported-files-from-original.md

```diff
@@ -0,0 +1,5 @@
+# This document will be a log to add the name of the files that we've added from the original repository: [Minimal-Mistakes](https://github.com/mmistakes/minimal-mistakes)
```

Comment:
The name of the file could be more explicit. Something like original files of the theme repository.